easyLight/js/components/detailContract/index.js

```diff
@@ -36,9 +35,9 @@ class DetailContract extends Component {
     header: null
   };
   render(){
+    console.log( this.props.navigation.state.params);
```

Comment:
Remove log for next PR please
Quita los logs para el siguiente Pull Request pls